sos/plugins/insights.py

```diff
@@ -13,16 +13,16 @@ class RedHatInsights(Plugin, RedHatPlugin):
     '''Collect config and log for Red Hat Insights
     '''
     plugin_name = 'insights'
-    packages = ['redhat-access-insights']
+    packages = ['redhat-access-insights', 'insights-client']
     profiles = ('system', 'sysmgmt')
-    conf_file = '/etc/redhat-access-insights/redhat-access-insights.conf'
+    conf_file = ('/etc/insights-client/insights-client.conf', '/etc/redhat-access-insights/redhat-access-insights.conf')
 
     def setup(self):
         self.add_copy_spec(self.conf_file)
         if self.get_option("all_logs"):
-            self.add_copy_spec("/var/log/redhat-access-insights/*.log*")
+            self.add_copy_spec("/var/log/insights-client/*.log*")
         else:
-            self.add_copy_spec("/var/log/redhat-access-insights/*.log")
+            self.add_copy_spec("/var/log/insights-client/insights-client.log")
 
```

Comment:
I thought we were going to keep both logs for now, too?

If there's a point when we think that the old package is safely deprecated (e.g. it was only available on RHEL7, never 8, and 7 is no longer receiving new sos releases) then we can remove this at that time.